### PR TITLE
User model

### LINKED ISSUE #8: User's
- [x] Testing
- [x] Model

### PR DESCRIPTION
**Pull request by sarynm12**
- user model created
- added validations
- spec created 
- added validations to spec, with all passing

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
```
Comment (nicorithner) [inline app/models/user.rb:1]: Nice. Looks good. Can lines 2, 3, and 5 be consolidated into a one liner?
`validates :username, :email, :phone, uniqueness: true, presence: true`

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -61,7 +61,7 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
-  shoulda::Matchers.configure do |config|
```
Comment (nicorithner) [inline spec/rails_helper.rb:64]: Sorry about that typo 🤦🏻‍♂️

**Review by nicorithner — CHANGES_REQUESTED** [review]
Looks great. If you have time take a look at the comments on the model file

File: app/models/user.rb (hunk 1 above)
Comment (sarynm12) [reply app/models/user.rb:1]: You bet! Good catch!

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  validates :username, :email, :phone, uniqueness: true, presence: true
```
Comment (nicorithner) [inline app/models/user.rb:2]: Looks great. Thank you for refactoring this

**Review by nicorithner — APPROVED** [review]

**Pull request merged**